code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -357,6 +357,8 @@
 	toggle_ai(AI_ON) // Run.
 	status_flags &= ~GODMODE
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_ABNORMALITY_BREACH, src)
+	//Run icon change proc.
+	update_icon()
```

Comment:
There's no reason to use update_icon in most situations, especially not on every breach call, as overrides of BreachEffect() are meant to have the icon updates in it.